Mirror/Runtime/Transport/EpicOnlineTransport/EOSSDKComponent.cs

```diff
@@ -412,21 +452,25 @@ public class EOSSDKComponent : MonoBehaviour {
         }
 
         private void OnApplicationQuit() {
-            if (EOS != null) {
+#if !UNITY_EDITOR
+            if (EOS != null)
+            {
                 EOS.Release();
                 EOS = null;
                 PlatformInterface.Shutdown();
             }
-
+#endif 
             // Unhook the library in the editor, this makes it possible to load the library again after stopping to play
 #if UNITY_EDITOR
             if (libraryPointer != IntPtr.Zero) {
-                Bindings.Unhook();
-
-                // Free until the module ref count is 0
-                while (FreeLibrary(libraryPointer) != 0) { }
+            Bindings.Unhook();
```

Comment:
Only formatting changed in this block and it didn't get better. I'd omit this chunk.